Added details about Hashing emails (#4379)
segmentio/segment-docs: src/connections/destinations/catalog/criteo-audiences/index.md

7. Click **Save Changes**.  

> 8. In the **Mappings** tab, click **New Mapping** and select **Add Users to Audience**. If you wish your emails to be hashed before sending to Criteo, please select "yes" from the dropdown under "Hash Emails". Otherwise, do not change any default settings. By Default, emails are not hashed before sending to Criteo - Criteo hashes emails before storing them.

Reviewer: ```suggestion
8. In the **Mappings** tab, click **New Mapping** and select **Add Users to Audience**. If you wish your emails to be hashed before sending to Criteo, please select "yes" from the dropdown under "Hash Emails". Otherwise, do not change any default settings. By Default, emails are not hashed before sending to Criteo - Criteo hashes emails before storing them. To hash emails before you send them to Criteo, select **yes** in the **Hash Emails** dropdown. By default, Criteo does not hash emails before sending them, but does hash emails before storing them.
```

Author: Hey @markzegarelli 
What I see above seems repetitive. Can we do this instead?

`In the **Mappings** tab, click **New Mapping** and select **Add Users to Audience**. To hash emails before you send them to Criteo, select **yes** in the **Hash Emails** dropdown. By default, emails are not hashed before you send them to Criteo; however, Criteo will hash the emails before storing them in our system. DO NOT change any other default settings. `

> 8. In the **Mappings** tab, click **New Mapping** and select **Add Users to Audience**. If you wish your emails to be hashed before sending to Criteo, please select "yes" from the dropdown under "Hash Emails". Otherwise, do not change any default settings. By Default, emails are not hashed before sending to Criteo - Criteo hashes emails before storing them.

Reviewer: ```suggestion
8. In the **Mappings** tab, click **New Mapping** and select **Add Users to Audience**. To hash emails before you send them to Criteo, select **yes** in the **Hash Emails** dropdown. By default, emails are not hashed before you send them to Criteo; however, Criteo will hash the emails before storing them in our system. DO NOT change any other default settings. 
```
